services/shhext/signal.go

```diff
@@ -19,8 +19,8 @@ func (h EnvelopeSignalHandler) EnvelopeExpired(hash common.Hash) {
 }
 
 // MailServerRequestCompleted triggered when mailserver send a ack with a requesID sent previously
```

Comment:
`MailServerRequestCompleted is triggered when mailserver sends an ack with a requesID sent previously`?

Reply:
thank you! that was based on an old version with different names. I'm updating it